tests/commands/test_build.py

```diff
@@ -611,10 +671,33 @@ def test_build_project_is_not_cwd(
         (EmitterMode.BRIEF, ["--verbosity=brief"]),
     ],
 )
+@pytest.mark.parametrize(
+    "charmcraft_yaml, metadata_yaml",
+    [
+        [
+            dedent(
+                """\
+                type: charm
+                """
+            ),
+            dedent(
+                """\
+                name: test-charm-name-from-metadata-yaml
+                summary: test summary
+                description: test description
+                """
+            ),
+        ],
+    ],
+)
 def test_build_bases_index_scenarios_provider(
     mode,
     cmd_flags,
     basic_project,
+    prepare_charmcraft_yaml,
+    prepare_metadata_yaml,
+    charmcraft_yaml,
+    metadata_yaml,
```

Comment:
```suggestion
@pytest.mark.parametrize("charmcraft_yaml", ["type: charm\n"],)
def test_build_bases_index_scenarios_provider(
    mode,
    cmd_flags,
    basic_project,
    prepare_charmcraft_yaml,
    charmcraft_yaml,
```

The metadata fixtures aren't getting used (and aren't needed).

Reply:
Following PR needs it.